libs/designer/src/lib/core/utils/outputs.ts

```diff
@@ -342,17 +349,26 @@ export const getUpdatedManifestForSchemaDependency = (manifest: OperationManifes
 
       const currentSchemaValue = getObjectPropertyValue(updatedManifest.properties.outputs, outputLocation);
 
+      const inputSchema = updatedManifest.properties?.inputs?.properties?.schema;
```

Comment:
just calling out that this logic is essentially hardcoding and assuming that all hybrid triggers will use `schema` as the name for the floating editor parameter

Reply:
yes, the properties of the floatingActionMenu go inside the `schema` object. It is specifically defined to add it in the `schema` of the inputs. For now that's by design anyway